Clear the :py:data:`sys.last_traceback`, :py:data:`sys.last_type`
and :py:data:`sys.last_value` attributes by deleting them instead
of setting them to ``None``. This better matches the behaviour of
the Python standard library.
